src/clusterfuzz/_internal/datastore/data_handler.py

```diff
@@ -19,6 +19,7 @@
 import re
 import shlex
 import time
+from typing import Any
```

Comment:
As a rule of thumb, I suggest avoid using `typing.Any` whenever a more specific type can be used.